beacon/beacon.cabal

```
@@ -0,0 +1,46 @@
+cabal-version:   3.0
+name:            beacon
+version:         0.1.0.0
+synopsis:
+  Benchmarking, exploration, and analisys of Consensus.
+
+description:
+  Tool that can be used to compare the performance of two consensus versions, through `ouroboros-consensus:db-analyser`.
+
+license:         Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+
+copyright:       2023 Input Output Global Inc (IOG)
+author:          IOHK Engineering Team
+maintainer:      operations@iohk.io
+category:        Network
+build-type:      Simple
+extra-doc-files:
+  CHANGELOG.md
+  README.md
+
+executable beacon
+  hs-source-dirs: app
+  main-is:        beacon.hs
+  default-language: Haskell2010
```

Given that this is a dev-only tool, we don't have to support GHC 8.10, so we could use GHC2021.
```suggestion
  default-language: Haskell2010
```